react/src/blueTheme.ts

```diff
@@ -138,7 +138,7 @@ export const blueTheme: ThemeOptions = {
                 '&$disabled': {
                     backgroundColor: ThemeColors.background.paper,
                     border: `1px solid ${Color(PXBColors.black[500]).alpha(0.12).string()}`,
-                    color: Color(PXBColors.black[500]).alpha(0.3).string(),
+                    color: Color(ThemeColors.text.primary).alpha(0.3).string(),
```

Comment:
What happens if we uncomment the `ThemeColors.text.disabled` and update it to 30%? Does that break a bunch of other stuff?

Reply:
This looks like it will work just fine, but this will need further investigation of all material ui components. I'm going to leave this as is for now and we can revisit this when we do the story to update the showcase with material components.